src/components/views/voip/CallViewForRoom.tsx

```diff
@@ -79,11 +79,50 @@ export default class CallViewForRoom extends React.Component<IProps, IState> {
         return call;
     }
 
+    onResizeStart = () => {
+        this.props.resizeNotifier.startResizing();
+    }
+
+    onResize = () => {
+        this.props.resizeNotifier.notifyTimelineHeightChanged();
+    }
+
+    onResizeStop = () => {
+        this.props.resizeNotifier.stopResizing();
+    }
```

Comment:
I'm a little surprised the linter isn't punishing you for lack of semicolons on these function definitions. Also they should all be `private`.

Reply:
Sorry for the missing `private`. Regarding the semicolons, I am pretty sure they aren't used in many places all over the codebase, so I kind of assumed they were not necessary... 

Btw, thanks for reviewing a bunch of my PRs lately!

Reply:
no worries on the member access - it's a thing that we should probably have a linter for but don't (event handlers rarely need to be public). 

For semicolons: I suspect no one set up the lint rule for it.